client/locations-list-map.js

```diff
@@ -1957,6 +1951,60 @@ const initGlobalAlert = () => {
   });
 };
 
+const initEmbedListEventListener = () => {
+  // Web toggle button
+  document.getElementById('js-ToggleListButton').addEventListener('click', toggleEmbedList);
+
+  // Mobile view toggle buttons
+  const mobileViewButtons = document.getElementsByClassName('emb-toggleButton');
+
+  for (let i = 0; i < mobileViewButtons.length; i++) {
+    mobileViewButtons[i].addEventListener('click', () => toggleMobileView(i));
+  }
+};
+
+const toggleEmbedList = () => {
+  const button = document.getElementById('js-ToggleListButton');
+  const container = document.getElementById('js-EmbedContainer');
+
+  if (container.classList.contains('list-open')) {
+    button.innerText = 'Open list';
```

Comment:
Can you add i18n for this? this is the wiki for it: https://github.com/findthemasks/findthemasks/wiki/Translation-Process

you'll need to add the strings to en.json, use $.i18n('string-key-name') in place of the hard coded strings, and then when you push up the PR the strings will be machine translated and a new PR will be opened against your current one that you can merge in

Reply:
I believe I've done it, worth getting another eye on it just in case, thanks Nate

Reply:
nice looks good. I'm noticing the button appears before the map and list do, is it possible to render after map load or list load? Not worth holding up merge but would be nice to have

Reply:
I can't for the life of me figure this out as the `display: block` gets added once everything has loaded, but the element is hidden a couple of seconds before it, can't see anything at a glance but like you said not a merge breaker.

Reply:
There are known performance issues of the list/map - they load increasingly slowly, especially on the larger US dataset. @ngottlieb has an issue for it on #1053 if anyone wants to investigate further :) thanks guys!